pkgs/tools/audio/beets/common.nix

```
@@ -55,7 +55,15 @@ let
   };
 
   basePlugins = lib.mapAttrs (_: a: { builtin = true; } // a) (import ./builtin-plugins.nix inputs);
-  allPlugins = lib.mapAttrs (n: a: mkPlugin { name = n; } // a) (lib.recursiveUpdate basePlugins pluginOverrides);
+  pluginOverrides' = lib.mapAttrs
+    (plugName: lib.throwIf
+      (builtins.hasAttr "deprecated" basePlugins.${plugName})
```

```suggestion
      basePlugins.${plugName}.deprecated or false
```

... otherwise it will throw if `deprecated` is explicitly set to `false`.

---

> ... otherwise it will throw if `deprecated` is explicitly set to `false`.

Good idea! Thanks again.